### ISSUE TITLE
Slashes being replaced with %2F

### ISSUE THREAD

**Issue by nicky-lenaers** [jspm/jspm-cli#1704]
In my ``jspm.conf.js`` (``v0.17.0-beta.12``), I have a custom ``map`` section set up like so:

```
map: {
    "nite": "app/ts_modules/nite",
    "three": "github:mrdoob/three.js@master/build/three.js",
    "three-orbitcontrols": "github:mrdoob/three.js@master/examples/js/controls/OrbitControls.js",
    "three-transformcontrols": "github:mrdoob/three.js@master/examples/js/controls/TransformControls.js"
}
```

Then, the logs from my Docker build show this:

```sh
Build Failed. Error Message: 
	 { [Error: Error on instantiate for source/app/ts_modules/nite/game/editor.ts at file:///src/source/app/ts_modules/nite/game/editor.ts
	Loading source/app/components/editor/Editor.ts
	Loading source/app/index.ts
	Error: ENOENT: no such file or directory, open '/src/build/public/jspm_packages/github/mrdoob/three.js@master%2Fexamples%2Fjs%2Fcontrols%2FOrbitControls.js.json'
	    at Error (native)
	Error loading /src/build/public/jspm_packages/github/mrdoob/three.js@master%2Fexamples%2Fjs%2Fcontrols%2FOrbitControls.js.json]
  originalErr: 
   { [Error: ENOENT: no such file or directory, open '/src/build/public/jspm_packages/github/mrdoob/three.js@master%2Fexamples%2Fjs%2Fcontrols%2FOrbitControls.js.json']
     errno: -2,
     code: 'ENOENT',
     syscall: 'open',
     path: '/src/build/public/jspm_packages/github/mrdoob/three.js@master%2Fexamples%2Fjs%2Fcontrols%2FOrbitControls.js.json' } }
```

The path of my custom maps replaces some slahes with ``%2F``. How can I resolve this?

**Comment by nicky-lenaers** [comment]
I'd like to add that this happens everytime I run a fresh ``jspm install``, if that helps debugging :+1:

**Comment by guybedford** [comment]
The workaround here would be to expand the map into `jspm_packages/github/mrdoob/three.js@master/build/...`.

I've added the ability to support this in https://github.com/jspm/jspm-cli/commit/c307dbb9a2b192c3fcab13d5c146ca3fd099e0e9.

**Comment by nicky-lenaers** [comment]
@guybedford Great! Thanks for the temporary workaround for now ;)

**Comment by guybedford** [comment]
Released in 0.17.0-beta.14.

**Comment by nicky-lenaers** [comment]
Awesome
